base/src/main/scala/typeclass/LiskovFunctions.scala

```diff
@@ -0,0 +1,13 @@
+package scalaz
+package typeclass
+
+import Liskov.<~<
+
+trait LiskovFunctions {
+  /** http://typelevel.org/blog/2014/03/09/liskov_lifting.html **/
+  def liftCvf[A, B, F[_]: Functor](a: A <~< B): F[A] <~< F[B] = a.asInstanceOf[F[A] <~< F[B]]
+
+  /**Subtyping is transitive */
+  def trans[A, B, C](f: B <~< C, g: A <~< B): A <~< C =
+    g.subst[λ[`-α` => α <~< C]](f)
```

Comment:
You may wish to provide antisymmetry too, `(A <~< B, B <~< A) => (A === B)`. One point in opposition: I don't think it can be written with `subst`.